pinot-core/src/main/java/org/apache/pinot/core/data/manager/realtime/LLRealtimeSegmentDataManager.java

```diff
@@ -85,7 +86,7 @@
 import org.apache.pinot.spi.stream.StreamPartitionMsgOffset;
 import org.apache.pinot.spi.stream.StreamPartitionMsgOffsetFactory;
 import org.apache.pinot.spi.stream.TransientConsumerException;
-import org.apache.pinot.spi.utils.CommonConstants.ConsumerState;
+import org.apache.pinot.spi.utils.CommonConstants;
```

Comment:
Any specific reason for this change?

Reply:
No, seems like IDE auto-refactor, reverted.